lib/manageiq/environment.rb

```diff
@@ -0,0 +1,108 @@
+require 'fileutils'
+require 'pathname'
+
+module ManageIQ
+  module Environment
+    APP_ROOT = Pathname.new(File.expand_path('../../..', __FILE__))
+
+    def self.ensure_config_files
+      config_files = {
+        "certs/v2_key.dev"        => "certs/v2_key",
+        "config/cable.yml.sample" => "config/cable.yml",
```

Comment:
Let's wait until after this PR gets in, then maybe you can do it...I'll ping you when this is in.